string type_name
int32 number_observations
bool accepted
float64 diameter
float64 height
geometry_msgs/PoseWithCovariance pose

